pkg/generate/app/cmd/newapp.go

```diff
@@ -623,6 +623,19 @@ func (c *AppConfig) buildPipelines(components app.ComponentReferences, environme
 			if c.NoOutput {
 				pipeline.Build.Output = nil
 			}
+			if err := pipeline.Validate(); err != nil {
```

Comment:
@bparees fixed a merge conflict here.

Validate comes after updating the build output.